### PR TITLE
Avoid deprecated notice when using symfony/config > 4.2

### LINKED ISSUE #289: Fix deprecations with symfony 4.2
| Q            | A
| ------------ | ---
| Bug?         | not yet
| New Feature? | no
| Version      | master

Now that symfony 4.2 is released, the ci fails because tests, while being green, trigger a couple of deprecation notices: https://travis-ci.org/php-http/HttplugBundle/jobs/462460147

We should adjust those things, ideally while staying backwards compatible with all supported versions.

### PR DESCRIPTION
**Pull request by odolbeau**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | fix #289 
| License         | MIT

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 The change log describes what is "Added", "Removed", "Changed" or "Fixed" between each release.
 
+## 1.13.2
```
Comment (dbu) [inline CHANGELOG.md]: ideally use "unreleased" here to avoid confusion when this is not released yet.

Comment (odolbeau) [reply CHANGELOG.md]: :+1:

**Comment by dbu** [comment]
thanks a lot!

**Comment by odolbeau** [comment]
Unfortunately, I don't have any idea... :/ It's the first time I see this deprecation warning.

**Pull request merged**